frontend/src/packages/projects/projects-api.ts

```diff
@@ -81,6 +81,28 @@ class ProjectsApi extends HttpApi {
 
     return await response.json<ProjectGetAllItemResponseDto>();
   }
+
+  public async updateProject(
+    projectId: number,
+    payload: ProjectRequestDto,
+  ): Promise<ProjectGetAllItemResponseDto> {
+    const response = await this.load(
+      this.getFullEndpoint(
+        configureString(ProjectsApiPath.$PROJECT_ID, {
+          projectId,
+        }),
+        {},
+      ),
```

Comment:
```suggestion
      this.getFullEndpoint(ProjectsApiPath.$PROJECT_ID, { projectId }),
```

Reply:
types is not compatible, getFullEndpoint expecting string